Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7.5-alpine3.10
+FROM python:3.8-alpine
```

Comment:
Is there any reason to pin to `3.8-alpine` and not to `3-alpine` (or at least to the most recent minor version, `3.13-alpine`)?

Reply:
Initially, the repository author used an image with a specific binding to the Python version. So there were ideological grounds for that. That's why I tried to install the minimum compatible version in the commit in accordance with the requirement of the used package. This is also an additional check for operability in the specified minimum Python version. By the way, if it weren't for this error, I wouldn't have found another one in [this PR](https://github.com/xbmc/addon-check/pull/285).

Reply:
This isn't the correct place for checking minimum Python version. That has to be done inside the [addon-check](https://github.com/xbmc/addon-check) repository.
@enen92 has to decide, if he wants to use the oldest possible or the (currently) newest version of Python.